fgpyo/util/metric.py

```diff
@@ -10,14 +10,25 @@
 
 The :class:`~fgpyo.util.metric.Metric` class makes it easy to read, write, and store one or metrics
 of the same type, all the while preserving types for each value in a metric.  It is an abstract
-base class decorated by `attr <https://www.attrs.org/en/stable/examples.html>`_, with attributes
-storing one or more typed values.
+base class decorated by `dataclassees <https://docs.python.org/3/library/dataclasses.html>`_, or
+`attr <https://www.attrs.org/en/stable/examples.html>`_, with attributes storing one or more typed
+values.
 
 Examples
 ~~~~~~~~
 
 Defining a new metric class:
 
+.. code-block:: python
+
+   >>> from fgpyo.util.metric import Metric
+   >>> import dataclasses
+   >>> @dataclasses.dataclass(frozen=True)
+   ... class Person(Metric["Person"]):
+   ...     name: str
+   ...     age: int
+
+or using attr:
 .. code-block:: python
```

Comment:
Does this format correctly?

Reply:
I'm not sure how to check. Is it easy to do?

Reply:
Nevermind, figured it out and fixed it.